### DISCUSSION TITLE
Add to readme instructions set "BLEAddrFilter 3" in console

### DISCUSSION THREAD

**Discussion by chlor3** [tony-fav/tasmota-blerry#59] [Ideas]
As a tasmota noob one of the things that tripped me up was my installation defaulted to "BLEAddrFilter 0" which filters out all but public addresses, not private like many of the devices we are trying to sniff.  Recommend adding that to the "How to" when attempting to get mac addresses.  

Firmware:                     tasmota32c3-bluetooth (12.1.1.2)
ESP32 HW:                   SwitchBot Plug Mini {W1901400)
Bluetooth Client HW:    SwitchBot MeterTH S1

**Comment by tony-fav** [comment]
@chlor3 I don't have any specific instructions on finding the mac addresses on purpose because there are an extreme many ways to do it and it would be impossible to maintain. I tend to use an app on my phone LightBlue but NRF Connect and several others are really the best. If I made any recommendations, I would say to do BLEAddrFilter 1

0 is for mac addresses that are "public" and "fixed"
1 is for mac addresses that are "private" and "fixed"
but 2 and 3 are for mac addresses that randomize. There's not much use trying to use those macs.
